tests/test_app/Makefile

```diff
@@ -111,8 +111,8 @@ compile: libcgra.so top.sv garnet_param.svh garnet_test.sv kernel.sv proc_driver
 # Run
 # -------------------------------------------------------------------
 .PHONY: run
-run:
-	xrun -R -sv_lib libcgra.so $(RUN_ARGS) $(APP_ARGS) +VCD_ON
+run: 
```

Comment:
Doesn't we need a flag to turn VCD on for debugging purpose?

Reply:
It's already handled in `aha glb` command.
If you add `--vcd` flag, $RUN_ARGS variable would be updated to dump vcd.
https://github.com/StanfordAHA/aha/blob/b56b280d47bdda5e545abf6101e9053cc038969c/aha/util/glb.py#L42
I think not dumping vcd by default would be better.